app/TabNavigation.tsx

```diff
@@ -16,8 +16,16 @@ export function TabNavigation(): ReactElement {
 
   return (
     <Wrapper>
-      <div style={{ display: "flex" }}>
+      <div
+        style={{
+          display: "flex",
+          backgroundColor: "black",
+          alignSelf: "stretch",
+          justifyContent: "center",
+        }}
+      >
         <LargeButton
+          width="150px"
```

Comment:
If you're going to accept this I think a number is safer, otherwise you can pass in "J if you jgykhyu hi J" and typescript won't complain

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/18/commits/431ec2d72dc81babb44550e8d79e33a8e1fa2690

Reply:
wait this isn't the fix for what I said

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/18/commits/9af70e7c16f27ddfa79a17596e3e5f37201c2c51